opentelemetry-sdk/src/opentelemetry/sdk/_configuration/__init__.py

```diff
@@ -112,21 +135,35 @@ def _import_tracer_provider_config_components(
 
 
 def _import_exporters(
-    exporter_names: Sequence[str],
-) -> Dict[str, Type[SpanExporter]]:
+    trace_exporter_names: Sequence[str],
+    log_exporter_names: Sequence[str],
+) -> Tuple[Dict[str, Type[SpanExporter]], Dict[str, Type[LogExporter]]]:
     trace_exporters = {}
+    log_exporters = {}
 
     for (
         exporter_name,
         exporter_impl,
     ) in _import_tracer_provider_config_components(
-        exporter_names, "opentelemetry_traces_exporter"
+        trace_exporter_names, "opentelemetry_traces_exporter"
     ):
         if issubclass(exporter_impl, SpanExporter):
             trace_exporters[exporter_name] = exporter_impl
         else:
             raise RuntimeError(f"{exporter_name} is not a trace exporter")
-    return trace_exporters
+
+    for (
+        exporter_name,
+        exporter_impl,
+    ) in _import_tracer_provider_config_components(
```

Comment:
Nit: Should we rename `_import_tracer_provider_config_components` to just `_import_config_components`?

Reply:
Fixed